schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -568,14 +569,34 @@ def update_conditions(self):
                 not hasattr(self, "_model_observatory")
                 or self._model_observatory.nside != self._scheduler.nside
             ):
+                # Get weather conditions from pickle.
+                wind_data = rubin_sim.site_models.ConstantWindData(
+                    wind_speed=self._conditions.wind_speed,
+                    wind_direction=self._conditions.wind_direction,
+                )
+                # Set seeing to fiducial site seeing.
+                seeing_data = rubin_sim.site_models.ConstantSeeingData(0.69)
+                # Create new MO instance.
                 self._model_observatory = ModelObservatory(
                     nside=self._scheduler.nside,
                     init_load_length=1,
+                    wind_data=wind_data,
+                    seeing_data=seeing_data,
                 )
 
+            # Code for tickets/PREOPS-4817
```

Comment:
In general, if code has previously been checked into git, just deleting it rather than commenting it is usually a better approach, although a text comment describing why one approach was taken rather than the other, maybe mentioning the commit where the other approach was taken, can be quite useful. Usually, "why" comments are more useful for posterity than "what" comments. (There are, of course, exceptions to this rule of thumb.)

(Just because I know this is good advice doesn't mean I actually always follow it myself, but I should.)

Reply:
Yes, `update_conditions` only gets called when the date is changed 👍

The commented code is actually for the next ticket related to this, [ticket 3817](https://jira.lsstcorp.org/browse/PREOPS-3847), allowing users to set weather conditions. (I realise now I've written the ticket number wrong.) I'll just remove the commented code and store in a file on my computer somewhere instead.